File: OOP-ClassesAndObjects_HristoChipev/Person.cs
using System;
namespace OOP_Objects_Classes
{
    public class Person
    {
        private string name;
        private int age;

        public Person(string name, int age)
        {
            this.age = age;
            this.name = name;
        }

        public string Name
        {
            get { return name; }
            set { name = value; }
        }

        public int Age
        {
            get { return age; }
            set { age = value; }
        }
    }
}

File: OOP_Classes&Objects_Hristiyana_Shopova/src/Family/Person.java
package Family;

public class Person {
    private String name;
    private int age;


    public void setAge(int age) {
        this.age = age;
    }

    public int getAge() {
        return age;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
    public Person(String nameC, int age)
    {
        this.name = nameC;
        this.age = age;
    }
}